.github/workflows/rust.yml

```diff
@@ -89,11 +91,10 @@ jobs:
       - name: Install deps
         run: cargo install cargo-apk
 
-        # TODO: Do we still need this?
         # Create folder without content to make the build succeed
       - run: |
           mkdir -p deps/android_openxr/arm64-v8a
-          cargo xtask prepare-deps --platform android
+          # cargo xtask prepare-deps --platform android
```

Comment:
this line should be removed completely. also the mkdir can be put inline to `run:` and remove "|"